api/Dockerfile-audit-trail-gw

```diff
@@ -4,7 +4,9 @@ WORKDIR /usr/src/app
 COPY . .
 
 RUN cd shared-modules && npm install && npm run build
-# TODO adjust
+
+WORKDIR /usr/src/app/audit-trail-gw
```

Comment:
The best practice require to use always the same directory so /usr/src/app is ok for both services